action.yml

```diff
@@ -27,15 +31,19 @@ runs:
   steps:
     - name: Environment setup
       shell: bash
+      env:
+        SETUPDUNEDIR: ${{ inputs.directory }}
       run: |
         echo "$HOME/.local/bin" >> "$GITHUB_PATH"
         echo "DUNE_CACHE_ROOT=$HOME/.cache/dune" >> "$GITHUB_ENV"
-    - name: Cache dune’s cache
-      uses: actions/cache@v4
+        echo "BUILDDIR=$PWD/$SETUPDUNEDIR/_build" >> "$GITHUB_ENV"
```

Comment:
Repeating the comment from the other location:

 https://github.com/mbarbin/catch-the-bunny/actions/runs/20335350373/job/58421827771

Unrelated, in this job I see a warning:
Warning: Invalid pattern '/home/runner/work/catch-the-bunny/catch-the-bunny/./_build'. Relative pathing '.' and '..' is not allowed.

> I don't know whether this is coming from the way the build dir is set by the setup-dune script, or something else.

I suspect this warning comes from the actions/cache job if SETUPDUNEDIR=".". No idea how important this is (just thought it would be nice to have the job warning free if possible).

Reply:
Confirming the warning is gone https://github.com/mbarbin/catch-the-bunny/actions/runs/20337530585/job/58427798686